packages/runtime/src/templates/server.ts

```diff
@@ -3,102 +3,105 @@ import { NodeRequestHandler, Options } from 'next/dist/server/next-server'
 
 import {
   netlifyApiFetch,
-  getNextServer,
   NextServerType,
   normalizeRoute,
   localizeRoute,
   localizeDataRoute,
   unlocalizeRoute,
 } from './handlerUtils'
 
-const NextServer: NextServerType = getNextServer()
-
 interface NetlifyConfig {
   revalidateToken?: string
 }
 
-class NetlifyNextServer extends NextServer {
-  private netlifyConfig: NetlifyConfig
-  private netlifyPrerenderManifest: PrerenderManifest
+const getNetlifyNextServer = (NextServer: NextServerType) => {
```

Comment:
[dust] consider declaring `getNetlifyNextServer` separately rather than enclosing the class, but it's just a style thing so no worries if you disagree!

Reply:
I'm not sure I understand:

I think I do need to enclose the class inside a function because to declare `NetlifyNextServer` class I need a reference to `NextServer` class. And I can't have actual guaranteed working `NextServer` class with either static import or with executing some code in module scope (as both eveluate eagerly, before we even have access to `app` directory which is needed to reliably get a hold of `next-server` module in `next`).

Maybe it would be doable with some prototype shenanigans to declare `NetlifyNextServer` class that doesn't extend `NextServer` initially and apply needed stuff once we get hold of `NextServer`, but I think this would actually be worse and more confusing than current approach?